src/components/AddResourceModal/AddWaterTap/AddWaterTap.jsx

```diff
@@ -81,17 +81,38 @@ const AddWaterTap = ({
       justifyContent="center"
       // width={isMobile ? '100%' : ''}
     >
-      <Typography
-        display="flex"
-        flexDirection="row"
-        alignItems="flex-end"
-        padding="0px 20px 10px"
-        height="88px"
-        backgroundColor="#5286E9"
-        color="common.white"
-      >
-        Add Water Resource
-      </Typography>
+      {isMobile ? (
```

Comment:
1. Typography should only contain text and font styling attributes, the rest should go to a `Stack`, `Box`, or `Grid` components
2. We should have one element and share the styles that are the same between mobile and desktop, the different attributes should be conditionally set with this flag